### PR TITLE
Fix null action diary events

### PR DESCRIPTION
**Pull request by LBHRShetty**
Fixed null values in action diary events

### CODE REVIEW

File: LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs [hunk 1]
```diff
@@ -308,6 +308,23 @@ public void WhenGivenTenancyRef_GetActionDiaryDetailsbyTenancyRef_ShouldReturnAl
...
 
+        [Theory]
+        [InlineData("78908/01", 1)]
+        [InlineData("12345/06", 1)]
+        public void WhenGivenTenancyRef_GetActionDiaryDetailsbyTenancyRef_ShouldReturnCompleteArrearsActions(string strTenancyRef,int num)
+        {
+            var insertedAction = InsertRandomActionDiaryDetails(strTenancyRef, num)[0];
+            var returnedAction = GetArrearsActionsByRef(strTenancyRef)[0];
+
+            Assert.Equal(insertedAction.TenancyRef.Trim(), returnedAction.TenancyRef.Trim());
```
Comment (StevenLeighton21) [inline LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs]: I think the domain object should be performing this trim?

File: LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs [hunk 2]
```diff
@@ -308,6 +308,23 @@ public void WhenGivenTenancyRef_GetActionDiaryDetailsbyTenancyRef_ShouldReturnAl
             Assert.Equal(numberOfExpectedActions, actions.Count);
         }
 
+        [Theory]
+        [InlineData("78908/01", 1)]
+        [InlineData("12345/06", 1)]
```
Comment (StevenLeighton21) [inline LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs]: Would like to see one of these tests inserting multiple action diary entries (unless the other test already covers this?).

File: LBHTenancyAPI/Gateways/UhTenanciesGateway.cs [hunk 3]
```diff
@@ -85,13 +85,13 @@ public List<ArrearsActionDiaryEntry> GetActionDiaryEntriesbyTenancyRef(string te
         {
             return conn.Query<ArrearsActionDiaryEntry>(
                 "SELECT " +
-                "tag_ref as TenancyRef, " +
-                "action_code as ActionCode, " +
+                "tag_ref as TenancyRef," +
```
Comment (StevenLeighton21) [inline LBHTenancyAPI/Gateways/UhTenanciesGateway.cs]: missing a space - not sure it should effect the running of the sql, but lets keep the spacing consistent :)

File: LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs (hunk 2 above)
Comment (LBHRShetty) [reply LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs]: Done! added multiple entries

File: LBHTenancyAPI/Gateways/UhTenanciesGateway.cs (hunk 3 above)
Comment (LBHRShetty) [reply LBHTenancyAPI/Gateways/UhTenanciesGateway.cs]: added space

File: LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs [hunk 4]
```diff
@@ -308,6 +308,23 @@ public void WhenGivenTenancyRef_GetActionDiaryDetailsbyTenancyRef_ShouldReturnAl
             Assert.Equal(numberOfExpectedActions, actions.Count);
         }
 
+        [Theory]
+        [InlineData("78908121/01", 4)]
+        [InlineData("12345121/06", 2)]
+        public void WhenGivenTenancyRef_GetActionDiaryDetailsbyTenancyRef_ShouldReturnCompleteArrearsActions(string strTenancyRef,int num)
```
Comment (zuren) [inline LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs]: Like the `InlineData` usage 😄

**Comment by zuren** [comment]
Have you tested this locally and seen good results @LBHRShetty ?

**Pull request merged**